config/telegram.js

```diff
@@ -0,0 +1,14 @@
+const {
+    TELEGRAM_API_KEY,
+    TELEGRAM_CHAT_ID,
+  } = process.env;
+  
+  const telegramKeys = [{

```

Comment:
It seems you're exporting an array of objects when you want an object.

As it is it won't work since you're importing it as an object down there.

Reply:
Fixed. Damm copy paste :D